src/main/java/dev/slimevr/vr/processor/skeleton/SkeletonConfigToggle.java

```diff
@@ -0,0 +1,38 @@
+package dev.slimevr.vr.processor.skeleton;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public enum SkeletonConfigToggle {
```

Comment:
This enum is duplicated, please look at `SkeletonConfigToggles.java`
https://github.com/SlimeVR/SlimeVR-Server/blob/main/src/main/java/dev/slimevr/vr/processor/skeleton/SkeletonConfigToggles.java